src/native/libs/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -139,6 +140,11 @@ if (GEN_SHARED_LIB)
     )
 endif()
 
+message(STATUS "OPENSSL_INCLUDE_DIR:    ${OPENSSL_INCLUDE_DIR}")
```

Comment:
I added these because it makes it much easier to debug what's going on, especially on CI